src/main/scala/com/horizon/exchangeapi/tables/Nodes.scala

```diff
@@ -175,6 +175,7 @@ object NodesTQ {
   def getAllNodesId(orgid: String): Query[Rep[String], String, Seq] = rows.filter(_.orgid === orgid).map(_.id)
   def getNodeTypeNodes(orgid: String, nodeType: String): Query[Nodes, NodeRow, Seq] = rows.filter(r => {r.orgid === orgid && r.nodeType === nodeType})
   def getNonPatternNodes(orgid: String): Query[Nodes, NodeRow, Seq] = rows.filter(r => {r.orgid === orgid && r.pattern === ""})
+  def getRegisteredNodesInOrg(orgid: String): Query[Nodes, NodeRow, Seq] = rows.filter(node => {node.orgid === orgid && node.publicKey =!= ""})
```

Comment:
Do we care about this query maybe grabbing "dead" nodes? Nodes that haven't had a heartbeat in a while? Or should we check that a node isn't in an error state? I guess I'm more asking for the org status route do we want to be showing only "healthy" registered nodes or is it just any node that's gone through the registration process?

Reply:
Good question. I can seeing it going either way. Since we aren't sure at this point, i'm leaning toward the simpler implementation, i.e. the way megha has currently coded it.